packages/api/src/external/carequality/document/query-documents.ts

```diff
@@ -80,3 +92,21 @@ export async function getDocumentsFromCQ({
     throw error;
   }
 }
+
+function buildInterrupt({
+  patientId,
+  cxId,
+  log,
+}: {
+  patientId: string;
+  cxId: string;
+  log: typeof console.log;
+}) {
+  return async (reason: string): Promise<void> => {
+    log(reason + ", skipping DQ");
+    await resetDocQueryProgress({
+      patient: { id: patientId, cxId },
+      source: MedicalDataSource.CAREQUALITY,
+    });
+  };
+}
```

Comment:
why aren't we setting it to failed just like DR?

also, shouldn't we make them match? ie `[Programming error]` + Sentry?

Reply:
The idea is that is expected to have a DQ triggered for a patient that doesn't have CQ direct enabled, even if the IHE GW is not available (sandbox), so this just marks the DQ for that patient @ CQ as empty, never happened.

Whereas the DR, it shouldn't execute, b/c it only does it DQ does.